locales/es.json

```diff
@@ -46,170 +46,170 @@
-   "terminating due to error": "terminando debido a un error",
+   "terminating due to error": "finalizando debido a un error",
    "There a a space in the version of {{versionKey}}. We corrected it for you ({{correction})": "Hay un espacio en la versión de {{versionKey}}. Lo corregimos por ti ({{correction})",
    "versions": "versiones",
    "versions in use": "versiones en uso",
    "downloading {{packageName}} {{version}}....": "descargando {{packageName}} {{version}}....",
-   "language to use (default: en)": "language to use (default: en)",
-   "download template": "download template",
-   "Start the Embark console": "Start the Embark console",
-   "When set, will print the gas cost for each contract deploy": "When set, will print the gas cost for each contract deploy",
-   "Graph will not include undeployed contracts": "Graph will not include undeployed contracts",
-   "Graph will not include functions": "Graph will not include functions",
-   "Graph will not include events": "Graph will not include events",
-   "ENS domain to associate to": "ENS domain to associate to",
-   "output usage information and help information": "output usage information and help information",
-   "Starting %s process": "Starting %s process",
-   "Starting Blockchain node in another process": "Starting Blockchain node in another process",
-   "Blockchain node is ready": "Blockchain node is ready",
-   "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}": "Storage process for {{storageName}} ended before the end of this process. Code: {{code}}",
-   "ipfs process started": "ipfs process started",
-   "Compiling...": "Compiling...",
-   "Building...": "Building...",
-   "gas at the price of": "gas at the price of",
-   "Wei, estimated cost:": "Wei, estimated cost:",
-   "using": "using",
-   "Embark is building, please wait...": "Embark is building, please wait...",
-   "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)": "instantiated js-ipfs object configured to the current environment (available if ipfs is enabled)",
-   "Initializing Embark Template...": "Initializing Embark Template...",
-   "Name your app (default is %s):": "Name your app (default is %s):",
-   "Unrecognized balance string \"%s\"": "Unrecognized balance string \"%s\"",
-   "{{className}} has a dependency on {{dependencyName}}": "{{className}} has a dependency on {{dependencyName}}",
-   ", but it is not set to deploy. It could be an interface.": ", but it is not set to deploy. It could be an interface.",
-   "Done. %s generated": "Done. %s generated"
+   "language to use (default: en)": "languaje a utilizar (Por defecto: es)",
```

Comment:
Typo on line 188: languaje  =》 lenguaje

Reply:
Great work on this! Can you fix this typo?

Reply:
Sure!, I'll push the changes

Reply:
@StatusSceptre, is @LaloGarza an authorized reviewer for this PR?